Automattic/jetpack
# AI Assistant: Change loading icon with Spinner component

Fixes #30716

## Proposed changes:

* Changes the loading icon with the Spinner component

The spinner can blend with the background depending on the theme. Since themes do not use CSS variables in a consistent way and we can't use fallback values because of [this](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/tools/postcss.config.js#L14), there is not much we can do about that other than setting a background color to the input regardless of theme.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add an AI Assistant block and make a request
* Check that the spinner is displayed while the request is underway

![2023-05-24_13-07-49](https://github.com/Automattic/jetpack/assets/8486249/a4e1deb1-e21f-4c1d-99da-17855b80dac8)
![2023-05-24_13-07-31](https://github.com/Automattic/jetpack/assets/8486249/6f4cbd28-b3a7-48e0-942f-ef816c3e2f8a)
![2023-05-24_13-08-17](https://github.com/Automattic/jetpack/assets/8486249/3c57c95e-6728-432a-bce8-47e074fd17c7)
